ledger/privpayouts.go

```diff
@@ -28,8 +28,24 @@ var PrivGeneData = []GenesisInitPayout{
 var PrivGeneDataFromImport = []string{
 	"0,RmBKxMWg4C4EMzYowisDEGSBwmnR6tPgjLs,50000,1,0",
 	"0,RmHFARk5xmoMNUVJ6UCHFiWQML1vxwUhw1b,1254.345,1,0",
+	"0,Rm7B35PAP24GkZW1Za2gKrSiigZQ7M46KfL,1000.23456,1,0",
+	"0,RmCM99PchggcoZWkMVBWqAMHBcsn3T6VetG,2000.23456,1,0",
 }
 
+/**
```

Comment:
line35~48:
please unify comment style and make comments meaningful

Reply:
just a note